docs/community/working-group.md

```diff
@@ -0,0 +1,189 @@
+# Working Groups
+
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
+
+The list, description, and membership of the existing Working Groups is available
+[here](working-group-definitions.md).
+
+## Terms
+
+* **Contributor** is used interchangeably within this doc as anyone participating in issues or
+  contributing code/RFCs/etc, regardless of their status with the PowerShell project.
+* **Repository Maintainers** are trusted stewards of the PowerShell repository responsible for
+  maintaining consistency and quality of PowerShell code.
+  One of their primary responsibilities is merging pull requests after all requirements have been fulfilled.
+  (Learn more about the Repository Maintainers [here](https://github.com/PowerShell/PowerShell/tree/master/docs/maintainers).)
+* The **PowerShell Committee** is responsible for the design and governance of the PowerShell project,
+  primarily by voting to accept or reject review-for-comment (RFC) documents.
+  (Learn more about the PowerShell Committee [here](https://github.com/PowerShell/PowerShell/blob/master/docs/community/governance.md#powershell-committee).)
+* A **Working Group** is a collection of people responsible for providing expertise on a specific
+  area of PowerShell in order to help establish consensus within the community and Committee.
```

Comment:
Another repetition of the description of a WG we should either keep synced or link to the canonical definition.